Expense/expenses.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose')
+const Schema = mongoose.Schema;
+
+const ExpenseSchema = new Schema({
```

Comment:
Not to be too pedantic 😆, but for readability's sake, we should be adding spaces inside curly braces, after commas and colons:

```JavaScript
const ExpenseSchema = new Schema({
    amount: {
        type: Number,
        required: true
    },
    description: {
        type: String,
        required: true
    },
    budget: {
        type: Schema.Types.ObjectId,
        ref: 'Budget'
    },
    category: {
        type: Schema.Types.ObjectId,
        ref: 'Category'
    }
}
```

https://github.com/airbnb/javascript#whitespace--comma-spacing
https://github.com/airbnb/javascript#whitespace--in-braces